src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -92,7 +93,9 @@ public async Task Handle(MigrateRepoCommandArgs args)
 
             try
             {
-                args.ArchiveUrl = args.AwsBucketName.HasValue()
+                args.ArchiveUrl = args.UseGithubStorage
+                    ? await UploadArchiveToGithub(args.GithubOrg, args.ArchivePath)
+                    : args.AwsBucketName.HasValue()
```

Comment:
nit: at this point I'd use `if..else` for readability.

Reply:
[85190e8](https://github.com/github/gh-gei/pull/1280/commits/85190e80419eb38939609a570ce854445ab51790) 👍🏽